zboss/include/addons/zcl/zb_zcl_thermostat_addons.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef ZB_ZCL_THERMOSTAT_ADDONS_H__
+#define ZB_ZCL_THERMOSTAT_ADDONS_H__
+
+#include "zboss_api.h"
+
+/*! \addtogroup zb_zcl_thermostat_addons */
+/*! @{ */
+
+/**@brief Thermostat Information cluster attributes according to ZCL Spec 6.3.2.2.1. */
+typedef struct {
+    zb_int16_t local_temperature;
+    zb_int16_t outdoor_temperature;
+    zb_uint8_t occupancy;
+    zb_int16_t abs_min_heat_setpoint_imit;
+    zb_int16_t abs_max_heat_setpoint_limit;
+    zb_int16_t abs_min_cool_setpoint_limit;
+    zb_int16_t abs_max_cool_setpoint_limit;
+    zb_int16_t PI_cooling_demand;
+    zb_int16_t PI_heating_demand;
+    zb_uint8_t HVAC_system_type_configuration;
+} zb_zcl_thermostat_info_attrs_t;
```

Comment:
This list goes to the `ATTRS_LIST_EXT` macro, thus should have the `attrs_ext_t` suffix.